rotkehlchen/chain/evm/decoding/interfaces.py

```diff
@@ -171,8 +171,64 @@ def _decode_vote_cast(self, context: DecoderContext) -> DecodingOutput:
 
 class ReloadableDecoderMixin(metaclass=ABCMeta):
 
```

Comment:
I would add a docstring for the usage that this interface is intended for since the arguments names and defenitions are too generic